blog/2021/08/general-survey.md

```diff
@@ -0,0 +1,268 @@
+
+![Trend of popularity of CI services over time](/assets/blog/2021-08-general-survey/trend-ci.png)
+
+## Contributors
+
+As of 2021-06-27, at least 4931 different users contributed to Julia packages
+hosted on GitHub, about 250 more compared to two months before.  Note that these
+figures are conservative as they don't include contributors identified as
+anonymous by GitHub API (for example because they committed changes using an
+email address not registered in any GitHub account), or any contributions to
+packages hosted on different websites, like GitLab.
+
+![Trend of numbers of contributors](/assets/blog/2021-08-general-survey/trend-contributors.png)
+
+The distribution of number of contributors to Julia packages shows that the vast
+majority of packages have a handful of contributors, with a median of 2
+contributors per package.  Only about 11% of packages have more than 10
+contributors, and about 1% have more than 40.
+
+![Distribution of numbers of contributors per package](/assets/blog/2021-08-general-survey/distribution-contributors.png)
+
+![Reverse cumulative distribution of number of contributors per package](/assets/blog/2021-08-general-survey/rev-cdf-contributors.png)
+
+<!-- The number of commits per package follow a log-normal distribution, with median -->
+<!-- of 59 commits.  About 35% of packages have at least 100 commits, while 2% have -->
+<!-- more than 1000. -->
+
+<!-- ![Distribution of numbers of commits per package](/assets/blog/2021-08-general-survey/distribution-commits.png) -->
+
+<!-- ![Reverse cumulative distribution of number of commits per package](/assets/blog/2021-08-general-survey/rev-cdf-commits.png) -->
+
+It's also interesting to see how likely Julia users are to contribute to
+multiple packages: 52% of users have contributed to a single package, a bit less than
+10% contributed to at least 10 different packages, while 26 super active people
+contributed to more than 100 packages.
+
+![Reverse cumulative distribution of number of packages contributed to](/assets/blog/2021-08-general-survey/rev-cdf-contributed.png)
+
+Finally, we looked at how many git commits users made: half of the community of
+the developers of Julia packages have at least 10 commits, 21% have at least 100
+commits, while 51 people made more than 2500 commits.
+
+![Reverse cumulative distribution of number of commits per contributor](/assets/blog/2021-08-general-survey/rev-cdf-contributors-commits.png)
+
+## Conclusions
+
+Here are our main findings:
+
+* most packages do follow good programming practices (documentation, testing,
+  continuous integration)! In particular:
+  * high fraction of packages with any tests (96%), and at least 10% of code as
+    tests (76%)
+  * almost all packages have setup for continuous integration (95%)
+  * documentation is not as popular; only 49% of packages have docs in the
+    repository.
+  * most packages either have a README or docs (88% have at least twenty lines
+	of one or the other)
```

Comment:
```suggestion
  * most packages either have a README or docs (88% have at least twenty lines of one or the other)
```
This isn't rendered properly otherwise:
![grafik](https://user-images.githubusercontent.com/6735977/127857833-85c660f9-8e21-4c8f-9fe3-99688f6b6269.png)

Reply:
Thanks!